frontend/apps/marketing/src/config/brand.ts

```diff
@@ -1,5 +1,5 @@
 export enum Brand {
-  CODE_DOT_ORG = 'code.org',
+  CODE_DOT_ORG = 'Code.org',
```

Comment:
This can be lowercase if it's a url (and not brand-y) since the others are too.
```suggestion
  CODE_DOT_ORG = 'code.org',
```

Reply:
This is a brand thing and not a URL

Reply:
Oh! In that case could do this?
```suggestion
  HOUR_OF_CODE = 'Hour of Code',
  CS_FOR_ALL = 'CSforAll',
```

Reply:
If these also used as cache keys and in URL path segments, but not intended as domain names or to be used as full URLs, does it make sense to shorten them and remove whitespace and punctuation? E.g. `cdo`, `hoc`, `csforall`?